src/GraphQL/Schemas/Primary/Substrate/Queries/GetTokensQuery.php

```diff
@@ -73,12 +72,11 @@ public function resolve($root, $args, $context, ResolveInfo $resolveInfo, Closur
         }
 
         return Token::loadSelectFields($resolveInfo, $this->name)
-            ->addSelect(DB::raw('CONCAT(collection_id,id) AS identifier'))
             ->when($collectionId = Arr::get($args, 'collectionId'), fn ($query) => $query->whereHas(
                 'collection',
                 fn ($query) => $query->where('collection_chain_id', $collectionId)
             ))
             ->when(Arr::get($args, 'tokenIds'), fn ($query) => $query->whereIn('token_chain_id', $args['tokenIds']))
-            ->cursorPaginateWithTotalDesc('identifier', $args['first']);
+            ->cursorPaginateWithTotalDesc('id', $args['first']);
```

Comment:
Previously, there was an issue where the collection chain IDs were not being sorted correctly in descending order, so I implemented a fix for that, unfortunately caused another problem. With your new fix it lost the default behaviour of sorting collection in descending order, is it fine?

Reply:
What was the issue with the sorting? Maybe there's another way to sort them properly.  It seems like using the `identifier` alias in the query here may have become out of scope when used as a subquery in another query.